### PR TITLE
Reduced the width of the login screen

### PR DESCRIPTION
**Pull request by Larspolo**
BEFORE:
image

AFTER:
image

### CODE REVIEW

File: src/vue/src/components/columns/SmallContent.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<template>
+    <b-row noGutters>
+        <b-col
+            cols="8"
+            offset="2"
+            lg="4"
+            offsetLg="4"
+            class="main-content"
```
Comment (mjvkeulen) [inline src/vue/src/components/columns/SmallContent.vue:8]: So we created another relative column layout (not used elsewhere right?), this does not solve the issue entirely: a mega widescreen will still show a too wide login screen.

Should we switch to using a max-width instead? E.g. 1000px. 

Tagging @engelhamer to decide on this.

Comment (Larspolo) [reply src/vue/src/components/columns/SmallContent.vue:8]: > So we created another relative column layout (not used elsewhere right?)

Yes. Not against

>  this does not solve the issue entirely: a mega widescreen will still show a too wide login screen.

It already caps

Comment (mjvkeulen) [reply src/vue/src/components/columns/SmallContent.vue:8]: Sorry, missed the main content class.

Layout doesn't seem to play nice with all viewports.

Too small on medium size screens (1000-1350]:
image

Should be full width on small screens < 576
image

Comment (Larspolo) [reply src/vue/src/components/columns/SmallContent.vue]: Added more steps

Comment (mjvkeulen) [reply src/vue/src/components/columns/SmallContent.vue]: Seems to fix the complaints with those dimensions, just this device is a bit awkward:

image

Now its more a NIT

**Review by mjvkeulen — APPROVED** [review]

File: src/vue/src/components/columns/SmallContent.vue [hunk 2]
```diff
@@ -0,0 +1,17 @@
+<template>
```
Comment (mjvkeulen) [inline src/vue/src/components/columns/SmallContent.vue:1]: [NIT] Component name does not fit the column pattern, small-content-single-column perhaps?

Comment (Larspolo) [reply src/vue/src/components/columns/SmallContent.vue:1]: `WideContent` is also single column

**Review by engelhamer — APPROVED** [review]

**Pull request merged**